packages/preview/zhongwenshu/0.0.1/typst.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "zhongwenshu"
+version = "0.0.1"
+entrypoint = "lib.typ"
+authors = ["zhineng <@zhineng>"]
+license = "MIT"
+description = "A package that make chinese style book."

```

Comment:
If possible descriptions should avoid starting with `A package` or similar. See [here](https://github.com/typst/packages/blob/main/docs/manifest.md#writing-a-good-description).